### PR TITLE
Fix cascading escaping on config descriptions

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- add kibana to the canvas
- open the config in the inspector have a look at the 'path' option and note the description
- close and open the config a few times (go back via the inspector header)
- look at the path description, it should not look like:
2017-05-05-115117_389x570_scrot

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7306/

File: jujugui/static/gui/src/app/components/inspector/config/config.js [hunk 1]
```diff
@@ -235,14 +235,15 @@ YUI.add('inspector-config', function() {
       var configElements = [];
 
       Object.keys(charmOptions).forEach((key) => {
-        var option = charmOptions[key];
+        // Clone the options so that we're not updating the stored options.
+        const option = this._clone(charmOptions[key]);
```
Comment (Lukewh) [inline jujugui/static/gui/src/app/components/inspector/config/config.js]: Decent

**Comment by Lukewh** [comment]
Thanks @huwshimi :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**